lib/utils/FileCache.mjs

```diff
@@ -34,7 +34,16 @@ export default class FileCache {
 		const useCheckSum = cacheStrategy === CACHE_STRATEGY_CONTENT;
 
 		debug(`Cache file is created at ${cacheFile}`);
-		this._fileCache = fileEntryCache.createFromFile(cacheFile, useCheckSum, undefined);
+
+		try {
+			this._fileCache = fileEntryCache.createFromFile(cacheFile, useCheckSum, undefined);
+		} catch {
+			debug(`Cache file might be corrupt, attempting to remove and recreate the cache file`);
+
+			rmSync(cacheFile);
```

Comment:
[suggestion] Not confident, but does giving the `force` option make sense?

```suggestion
			rmSync(cacheFile, { force: true });
```

Reply:
https://nodejs.org/api/fs.html#fsrmsyncpath-options
> When true, exceptions will be ignored if path does not exist. 

I think it is indeed good to add this, thank you for the suggestion.

Then we avoid throwing with a more confusing error when the initial throw was unrelated to an existing cache file and no cache file exists.